operations/aws-security-roles/infosec-security-audit-incident-response-guardduty-roles-cloudformation.yml

```diff
@@ -48,101 +48,36 @@ Resources:
             Principal:
               AWS: arn:aws:iam::371522382791:root
             Action: sts:AssumeRole
+      ManagedPolicyArns:
+        - arn:aws:iam::aws:policy/SecurityAudit
```

Comment:
This managed role now includes requests to `APIGatewayAccess`, which was not originally included. Not sure if that is needed, but considering it only `GET` actions should be fine.